README.md

```
@@ -0,0 +1,1368 @@
+<td></td>
+</tr>
+<tr>
+<td>New York Stock Exchange</td>
+<td>XNYS</td>
+<td>NYSE market holidays (used by all other US-exchanges, including NASDAQ, etc.)</td>
+<td></td>
+</tr>
+</tbody>
+</table>
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)